Sources/Sentry/SentryProfiler.mm

```diff
@@ -219,16 +228,32 @@
         }
         const auto relativeTimestamp = getDurationNs(transaction.startSystemTime, timestamp);
 
+        const auto elapsed = sentry_stringForUInt64(relativeTimestamp);
```

Comment:
same here - this is more likely to crash than it is to return a nil value (e.g. if malloc fails)